src/EventTimer.ts

```diff
@@ -228,7 +228,7 @@ export class EventTimer {
 		}
 	}
 
-	trackInGA(eventName: string, label = ''): void {
+	private trackInGA(eventName: string, label = ''): void {
```

Comment:
This is super minor and feel free to ignore:

Can we list the private & public methods together?

Reply:
Good call